README.md

```diff
@@ -64,6 +64,10 @@ patches `require` + `autoload` to use these absolute paths, thereby avoiding hav
 
 Problem outlined in this [talk](https://www.youtube.com/watch?v=kwkbrOwLsZY)
 
+## Troubleshooting
+
+If you're experiencing problems with loading your application and you're unable to successfully run `Bootscale.regenerate`, delete the `tmp/bootscale` folder.
```

Comment:
That's really surprising. The cache files use a hash of the `$LOAD_PATH` in their name, so you shouldn't end up with a corrupted cache. Too bad you deleted it, I would have loved to take a peek at it to understand what happened.